packages/components/app/styles/components/table.scss

```diff
@@ -107,6 +124,7 @@ $hds-table-border-color: var(--token-color-border-primary);
       &:focus-visible {
         color: var(--token-color-foreground-strong);
         border-color: var(--token-color-focus-action-external);
+        outline: 0; // I'm not sure what to do here because we'll need our outline style but you can't change the outline radius
```

Comment:
cc @didoo I am not sure how to approach this, since we can't control the radius of the outline.